homeassistant/components/lyric/climate.py

```diff
@@ -166,7 +170,11 @@ def __init__(
     @property
     def supported_features(self) -> int:
         """Return the list of supported features."""
-        return SUPPORT_FLAGS
+        if self.device.changeableValues.thermostatSetpointStatus:
```

Comment:
Will this be constant for the lifetime of the entity? Supported features shouldn't change during entity lifetime.

Reply:
It should be constant afik. Only LCC devices will have the thermostatSetpointStatus variable defined, so only they should have the SUPPORT_PRESET_MODE flag set. If the TCC (like the Lyric) have the flag set it's no big deal, it just signals the frontend to provide the ability to change presets, which for the Lyric does nothing.